packages/hub/services/serializers/email-card-drop-request-serializer.ts

```diff
@@ -7,6 +8,18 @@ interface EmailCardDropRequestClaimStatus {
 }
 
 export default class EmailCardDropRequestSerializer {
+  serialize(model: EmailCardDropRequest): JSONAPIDocument {
+    return {
+      data: {
+        type: 'email-card-drop-requests',
+        id: model.id,
+        attributes: {
+          'owner-address': model.ownerAddress,
+        },
+      },
+    } as JSONAPIDocument;
```

Comment:
Dani helped me realise this casting is unnecessary if we have the return type specified and the data structure fits a `JSONAPIDocument`

Reply:
hah I even read that and still copypasted 🙃 removed now, thanks.